docker-compose.local.yml

```diff
@@ -8,9 +8,9 @@ services:
     ports:
       - 5432:5432
     environment:
-      - POSTGRES_USER=hello_zm
-      - POSTGRES_PASSWORD=hello_zm
-      - POSTGRES_DB=hello_zm_dev
+      - POSTGRES_DB
```

Comment:
I do not think we need it. It is in `.env`

Reply:
Docker isn't automatically passing all env variables to all containers